File: sandbox/enchanter/enchanter-scripting/enchanter-ruby/src/examples/stats.rb
server = "www.twdata.org"
user = "mrdon"
prompt = ":~>"

$conn.connect(server, user);
$conn.setTimeout(20000);
$conn.waitFor(prompt);
$conn.sendLine("top");
puts "==== Top ====\n"
puts $conn.getLine()
puts $conn.getLine()
puts $conn.getLine()
puts $conn.getLine()
$conn.send("^C")

$conn.waitFor(prompt)
puts "\n==== Disk Usage ====\n"
$conn.sendLine("df")
mux = java.lang.reflect.Array.newInstance(java.lang.String, 2);
mux[0] = "\r\n"
mux[1] = prompt
while (true) 
	id = $conn.waitForMux(mux)
	if id == 0 
		puts $conn.lastLine()
	else
		retry
	end
end
$conn.disconnect()

File: sandbox/enchanter/enchanter-scripting/enchanter-python/src/examples/stats.py
server = "www.twdata.org"
user = "mrdon"
prompt = ":~>"

conn.connect(server, user);
conn.setTimeout(20000);
conn.waitFor(prompt);
conn.sendLine("top");
print "==== Top ====\n"
print conn.getLine()
print conn.getLine()
print conn.getLine()
print conn.getLine()
conn.send("^C")

conn.waitFor(prompt)
print "\n==== Disk Usage ====\n"
conn.sendLine("df")
while 1:
    id = conn.waitForMux(["\r\n", prompt])
    if id == 0 :
        print conn.lastLine()
    else:
        break
conn.disconnect()
